st2actions/tests/unit/policies/test_concurrency.py

```diff
@@ -0,0 +1,103 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import copy
+
+import eventlet
+import mock
+import six
+
+import st2actions.bootstrap.runnersregistrar as runners_registrar
+from st2actions.runners.localrunner import LocalShellRunner
+from st2common.constants import action as action_constants
+from st2common.models.api.action import ActionAPI
+from st2common.models.api.policy import PolicyTypeAPI, PolicyAPI
+from st2common.models.db.action import LiveActionDB
+from st2common.persistence.action import Action, LiveAction
+from st2common.persistence.policy import PolicyType, Policy
+from st2common.services import action as action_service
+from st2common.transport.liveaction import LiveActionPublisher
+from st2common.transport.publishers import CUDPublisher
+from st2tests import DbTestCase
+from st2tests.fixtures import executions
+from st2tests.fixturesloader import FixturesLoader
+from tests.unit.base import MockLiveActionPublisher
+
+
+TEST_FIXTURES = {
+    'policytypes': [
+        'policy_type_1.yaml'
+    ],
+    'policies': [
+        'policy_1.yaml'
+    ]
+}
+
+PACK = 'generic'
+LOADER = FixturesLoader()
+FIXTURES = LOADER.load_fixtures(fixtures_pack=PACK, fixtures_dict=TEST_FIXTURES)
+NON_EMPTY_RESULT = 'non-empty'
+
+
+def mock_run(action_parameters):
+    eventlet.sleep(10)
+    return (action_constants.LIVEACTION_STATUS_SUCCEEDED, NON_EMPTY_RESULT, None)
+
+
+@mock.patch.object(
+    LocalShellRunner, 'run',
+    mock.MagicMock(side_effect=mock_run))
+@mock.patch.object(
+    CUDPublisher, 'publish_update',
+    mock.MagicMock(return_value=None))
+@mock.patch.object(
+    CUDPublisher, 'publish_create',
+    mock.MagicMock(side_effect=MockLiveActionPublisher.publish_create))
+@mock.patch.object(
+    LiveActionPublisher, 'publish_state',
+    mock.MagicMock(side_effect=MockLiveActionPublisher.publish_state))
+class ConcurrencyPolicyTest(DbTestCase):
```

Comment:
This should inherit from EventletTestCase since this is an eventlet test case. Only then, it will work if you run this standalone with nosetests I think.